pkg/plugins/policies/meshhealthcheck/api/v1alpha1/validator_test.go

```diff
@@ -271,6 +283,29 @@ violations:
   - field: spec.to[0].default.http.expectedStatuses[1]
     message: must be in inclusive range [100, 599]`,
 			}),
+			Entry("cannot use MeshExternalService with other type than Mesh", testCase{
+				inputYaml: `
+targetRef:
+  kind: MeshService
```

Comment:
kind MeshService will be deprecated for top-level, let's use MeshSubset for this test